www/src/components/navigation/leftSidebar.astro

```diff
@@ -40,8 +40,8 @@ if (langCode !== "en") {
           .flat()
           .find(
             ({ link: nativeLink }) =>
-              // trailing slash + language code = 3 characters
-              nativeLink.slice(3) === item.link.slice(3),
+              // trailing slash + language code
+              nativeLink.slice(langCode.length + 1) === item.link.slice(3),
```

Comment:
does this work with the nativeLink being sliced to a dynamic length and item being sliced to 3?

Reply:
yes
```js
const code = 'zh-cn';
'/zh-cn/usage/first-steps'.slice(code.length + 1) === '/en/usage/first-steps'.slice(3);
```